reference/enterprise-requirements.rst

```diff
@@ -58,5 +59,37 @@ applies:
 |                       |                       | services              |
 +-----------------------+-----------------------+-----------------------+
 
-[note] **Note:** Three nodes are required for multi-node operation.
-[/note]
+.. note ::
+    Three nodes are required for multi-node operation.
+
+Role Based Minimum Memory Sizing
+--------------------------------
+
+For multi-node deployments the following minimum component memory sizing may be
+used to size each node based on the roles it hosts within the deployment:
+
++-----------------+----------+---------+--------------+----------+--------+-----------------+
+| Component       | Control  | Compute | Storage      | Sunbeam  | Client | Juju-Controller |
```

Comment:
Compared to the "Install Canonical OpenStack using Canonical MAAS" How-To guide, this is missing specs for "Governor" and "MAAS" node types. I think I guess where this is coming from, but if the OpenStack team is not in a position to judge on those minimum, the How-To needs to be updated to point to the documentation of other products and provide a note so that the specs of "Governor" need to be big enough to accommodate hosting all the VMs that it's supposed to host, for example. If we could get such a note added to this PR for the MAAS How-To for now that would be great and we'll stay consistent.